d2/src/main/java/com/linkedin/d2/xds/XdsClientImpl.java

```diff
@@ -614,6 +625,10 @@ private void handleResponse(DiscoveryResponseData response)
       {
         return;
       }
+      if (!_responseReceived && response.getControlPlaneIdentifier() != null)
+      {
+        _log.info("Successfully established stream with ADS server: {}", response.getControlPlaneIdentifier());
```

Comment:
Can we also test that this is working correctly?

Reply:
I did test it, it works 👍